src/App.tsx

```diff
@@ -1,61 +1,125 @@
+import React, { useState } from 'react';
 import './App.scss';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+function getUserById(userId: number) {
+  return usersFromServer.find(user => user.id === userId)
+    || null;
+}
 
 export const App = () => {
+  const [titleValue, setTitleValue] = useState('');
```

Comment:
It's better to have more descriptive initial state variables. For instance, `selectedUserId` instead of `selectValue` can give a clearer idea about the purpose of this state variable.